release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1050,6 +1050,18 @@ The Special Resource Operator (SRO) includes a consistent log output format with
 
 Before this update, SRO did not support connecting to registries in disconnected environments. This release provides support for disconnected environments for which the driver container is hosted in a registry outside of the {product-title} cluster.
 
+[id="ocp-4-11-RAN-disabling-chronyd"]
+====  Disable chronyd in the PolicyGenTemplate CR
+
+You must disable `chronyd` if you update to {product-title} 4.11 from earlier versions. You can disable `chronyd` in the `[service]` section under `.spec.profile.data` in the `TunedPerformancePatch.yaml` file, which is included in the group `PolicyGenTemplate` CR. Add the following line in `TunedPerformancePatch.yaml`:
```

Comment:
To set context for the user, it might help to add a reference to the [section in the main docs](http://file.emea.redhat.com/amolnar/TELCODOCS-628/scalability_and_performance/ztp-vdu-validating-cluster-tuning.html#recommended-cluster-kernel-configuration), where this configuration is being discussed.